config/default.yml

```diff
@@ -174,6 +174,15 @@ Bundler/GemComment:
   IgnoredGems: []
   OnlyFor: []
 
+Bundler/GemFilename:
+  Description: 'Enforces the filename for managing gems.'
+  Enabled: true
+  VersionAdded: '<<next>>'
+  EnforcedStyle: 'Gemfile'
+  SupportedStyles:
+    - 'Gemfile'
+    - 'gems.rb'
```

Comment:
This configuration section should have an `Include` key to limit what's scanned by the cop (otherwise, it'll scan every single file, which is unperformant and might end up with false positives, for instance if someone has a `lib/gems.rb` file).